EIPS/eip-3521.md

```diff
@@ -0,0 +1,105 @@
+---
+eip: 3521
+title: Reduce access list cost
+author: Matt Garnett (@lightclient)
+discussions-to: https://ethereum-magicians.org/t/eip-3521-reduce-access-list-cost/6072
+status: Draft
+type: Standards Track
+category: Core
+created: 2021-04-15
+requires: 2028, 2718, 2929, 2930
```

Comment:
I think we can rely on 2718 and 2929 being transitively included via 2930 so we could narrow this to only 2028, 2930 I think.